app/helpers/application_helper.rb

```diff
@@ -17,7 +17,7 @@ def order_detail_description(order_detail) # TODO: deprecate in favor of "_as_(h
     order_detail_description_as_html(order_detail)
   end
 
-  def order_detail_description_as_html(order_detail)
+  def order_detail_description_as_html(order_detail) # TODO: deprecate in favor of OrderDetailPresenter#description_as_html
```

Comment:
Can this code be replaced with a direct call to `OrderDetailPresenter#description_as_html`?